blockchain/indexers/addrindex.go

```diff
@@ -30,7 +30,14 @@ const (
 
 	// addrKeySize is the number of bytes an address key consumes in the
 	// index.  It consists of 1 byte address type + 20 bytes hash160.
-	addrKeySize = 1 + 20
+	//addrKeySize = 1 + 20
+
+	// addrKeySize is the number of bytes an address key consumes in the
+	// index.  It consists of 1 byte address type + 32 bytes hash256.
+	// TODO: Since we updated the size, please note that we need to either:
+	// a) detect old indexes and force them to be rebuilt
+	// b) write a migration of the existing index
+	addrKeySize = 1 + 32
```

Comment:
Should this be increased further to support larger valid cashaddr addresses?

Reply:
I assume this is referencing the size of the payload? If so, some examples for reference:

https://gitlab.com/cash-accounts/specification/-/blob/master/SPECIFICATION.md#payment-data-types